Automattic/pocket-casts-ios
# Remove the newSearch feature flag

- Removes the `newSearch` feature flag
- Removes the unused `DiscoverPodcastSearchResultsController` and `PodcastListSearchResultsController` classes/xibs
- Updates the `newSearchResultsController` to `searchResultsController`

## To test
1. Enable the tracks logging feature flag
2. Launch the app
3. Go to the Podcasts tab
4. Perform a search
5. ✅ Verify the search is performed
6. ✅ Verify you see `🔵 Tracked: search_performed ["source": "podcasts_list"]` in console
7. Subscribe to some podcast results
8. ✅ Verify you are subscribed and you see `🔵 Tracked: podcast_subscribed ["source": "podcasts_list", "uuid": "XX"]`
9. Tap on a podcast result
17. ✅ Verify it opens the podcast details
10. Tap Show All
11. ✅ Verify the list loads
12. ✅ Verify tapping on a podcast opens the details
13. ✅ Verify tapping the + button subscribes you
14. Go back
15. Tap on an episode result
16. ✅ Verify it opens the episode details
17. Tap Show All
18. Tap on an episode result
16. ✅ Verify it opens the episode details
17. Repeat the steps on the discover tab

## Checklist
